src/api/todos.ts

```diff
@@ -0,0 +1,44 @@
+import { Todo } from '../types/Todo';
+import { client } from '../utils/fetchClient';
+import { Filter } from '../types/Filter';
+
+export const USER_ID = 6657;
+export const links = ['All', 'Active', 'Completed'];
+
+export const getTodos = (userId: number) => {
+  return client.get<Todo[]>(`/todos?userId=${userId}`);
+};
+
+export const postTodo = (userId: number, todo: {}) => {
+  return client.post(`/todos?userId=${userId}`, todo);
+};
+
+export const deleteTodo = (todoId: number) => {
+  return client.delete(`/todos/${todoId}`);
+};
+
+export const filterTodos = (
+  todos: Todo[],
+  filterParam: Filter,
+) => {
+  const visibleTodos = [...todos];
+
+  switch (filterParam) {
+    case Filter.Active:
+      return visibleTodos.filter(todo => !todo.completed);
+    case Filter.Completed:
+      return visibleTodos.filter(todo => todo.completed);
+    default:
+      return visibleTodos;
```

Comment:
better to add Filter.All case, so then we can quickly understand and increase our Filter enum

Reply:
not fixed